app/build.gradle.kts

```diff
@@ -94,6 +95,10 @@ dependencies {
     implementation("androidx.preference:preference:1.2.1")
     implementation("com.google.android.material:material:1.12.0")
     implementation("com.github.yalantis:ucrop:2.2.9")
+    implementation("androidx.tracing:tracing:1.2.0")
+    androidTestImplementation("androidx.test.ext:junit:1.2.0-beta01")
+    androidTestImplementation("androidx.test:rules:1.6.0-beta01")
+    androidTestImplementation("androidx.test.espresso:espresso-contrib:3.6.0-beta01")
```

Comment:
I'm seeing some duplicates here. All three of these seem to also be listed under the "Espresso" header. So I think those could be removed here?